styles/globals.css

```diff
@@ -67,3 +67,9 @@ footer a {
   min-height: 92vh;
   padding-bottom: 50px;
 }
+.d-none{
```

Comment:
Add a space between classname and curly brace

Reply:
Done